CHANGELOG.md

```diff
@@ -2,17 +2,18 @@
 
 ## New Features
 
-* Add check timeouts
-* Add support for check skip
+* You can configure timeout for checks now: just add `timeout: <seconds>` to a check in a ruleset.
```

Comment:
It is a bit more complicated: There is also a default timeout and CLI option (`--timeout`) to overwrite the timeouts. Is it worth mentioning that?

Reply:
Yup.

Reply:
Thanks.